### ISSUE TITLE
Animation import

### ISSUE THREAD

**Issue by ShellyHerself** [gbMichelle/Blendkrieg#12]
Make it so that animations can be properly imported from the jma class. Also have compressed animations only import the frames that are actually keyed.

**Comment by marcospampi** [comment]
I made progress in my fork, though the armature generation must be reworked

**Comment by ShellyHerself** [comment]
I'll check out your progress soon. The armature generation is hard because of some constraints posed by Halo's format. Mainly the issue where Blender bones have their children parented to their tail rather than their head. I'm open to suggestions for that

**Comment by marcospampi** [comment]
Yep, the way to achieve this would be to just discard the armature adjustment and leave it like nodes:
![Schermata del 2020-08-23 19-46-13](https://user-images.githubusercontent.com/25354450/90988158-05e2f500-e591-11ea-98fa-fea9778a3611.png)

**Comment by ShellyHerself** [comment]
I'd have to see how possible this is. I also want to see if armatures can possibly be saved. Because from my knowledge you can't skin a mesh to objects, only to armature bones.

**Comment by ShellyHerself** [comment]
Actually. I know exactly how this could work with armatures with just a tiny bit of math. Keep doing what you're doing when the time comes we'll work together to get it more in line with the other code and have it work with armatures.
